### DISCUSSION TITLE
what is this options used for?

### DISCUSSION THREAD

**Discussion by alaric8** [alefragnani/vscode-project-manager#760] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/7011d9ad-e9f8-41ae-8642-edfceab6c9a9)

**Comment by alefragnani** [comment, marked as answer]
Hi @AlaricWeb ,

This has no real purpose today, and will probably be removed in upcoming release.

Historically speaking, this was created with the intention to support multi-root projects scenarios, before multi-root projects were added natively in VS Code. When they published the plans to add it, I halted the implementation, and looked closely on the progress. Since then, these setting has no real purpose.

I’m planning to remove it in the next release, once the structure will be updated to support Profiles (#281).

Hope this helps

**Reply by alaric8** [reply]
Ok,thanks
